core/keys/site.go

```diff
@@ -58,5 +58,6 @@ const (
 	BatteryModeExternal = "batteryModeExternal"
 
 	// smart charging
-	SmartCostAvailable = "smartCostAvailable" // smart cost available
+	SmartCostAvailable   = "smartCostAvailable"   // smart cost available
+	SmartFeedinAvailable = "smartFeedinAvailable" // smart feed-in available
```

Comment:
Name here should match https://github.com/evcc-io/evcc/pull/21813/commits/97c727e8f59101e93a5cdf699995d5a979cb50f5#diff-84af0d1e3d30ba02a34b25442bda7c9d5c642c443839c900186255eaeecf693dR50

Reply:
I deliberately did not add the `Priority` because this values should be the same for all smartFeedin features (`smartFeedinPriorityLimit` (here), `smartFeedinDisableLimit` (later))

Reply:
I'd prefer the values to be consistent throughout the code. They are for `SmartCost`, but not for `smartFeedinPriority`. Since `SmartCost`doesn't follow the "what will happen" naming convention it can't be used as a valid comparison. Renaming _that_ was not desirable according to discussion.

Reply:
> We need the distinction since the latter will depend on tariff and device, the former only on tariff!

Yes, this is the part I've missed. I thought dynamic feedin is the same (and only) requirement for both.